modules/psap-special-resource-operator-using-CRD.adoc

```diff
@@ -0,0 +1,404 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-special-resource-operator.adoc
+
+:_content-type: PROCEDURE
+[id="deploy-simple-kmod-using-CRD"]
+= Building and running the simple-kmod SpecialResource for a hub and spoke topology
+
+You can use the Special Resource Operator (SRO) on a hub and spoke deployment in which Advanced Cluster Management connects a hub cluster to one or more managed clusters.
+This example shows how the Special Resource Operator builds driver containers in the hub, watching hub cluster resources to identify OCP versions for the helm charts that it uses to create resources which it delivers to spokes.
```

Comment:
I'm sorry, I don’t understand this description. Did you mean: 
```suggestion
This example shows how the Special Resource Operator builds driver containers in the hub. SRO watches the hub cluster resources to identify OCP versions for the helm charts that it uses to create resources which it delivers to spokes.
```

Reply:
Done